snapshot_manager/snapshot_manager/snapshot_manager.py

```diff
@@ -280,16 +280,15 @@ def check_todays_builds(self) -> None:
             # But if a package was re-build and failed, the old request ID for that chroot is invalid.
             # To compensate for this scenario that we saw on April 1st 2024 btw., we're gonna
             # delete any request that has a different set of Copr build IDs associated with it.
-            if chroot in requests:
-                recovered_request = requests[chroot]
-                if set(recovered_request.copr_build_ids) != set(current_copr_build_ids):
+            if request in requests:
```

Comment:
Where is `request` being assigned? I don't see this variable being used anywhere in this function.

Reply:
You're right.